public/app/features/explore/AddToDashboard/addToDashboard.ts

```diff
@@ -80,6 +80,9 @@ function getPanelType(queries: DataQuery[], queryResponse: ExplorePanelData) {
     if (queryResponse.traceFrames.some(hasQueryRefId)) {
       return 'traces';
     }
+    if (queryResponse.customFrames.some(hasQueryRefId)) {
```

Comment:
This should also be covered in addToDashboard.test.ts

Reply:
This should now be covered by the tests.